contrib/cirrus/setup_environment.sh

```diff
@@ -142,6 +142,26 @@ case "$CI_DESIRED_NETWORK" in
     *)          die_unknown CI_DESIRED_NETWORK ;;
 esac
 
+# Database: force SQLite or BoltDB as requested in .cirrus.yml.
+# If unset, will default to BoltDB.
+# shellcheck disable=SC2154
+case "$CI_DESIRED_DATABASE" in
+    sqlite)
+	warn "Forcing PODMAN_DB=sqlite"
+	echo "PODMAN_DB=sqlite" >> /etc/ci_environment
+	;;
+    boltdb)
+	warn "Forcing PODMAN_DB=boltdb"
+	echo "PODMAN_DB=boltdb" >> /etc/ci_environment
+	;;
+    "")
+	warn "Using default Podman database"
+	;;
+    *)
+	die_unknown CI_DESIRED_DATABASE
+	;;
+esac
+
```

Comment:
Minor: The above block mixes tab-characters and spaces.  Next time you need to update, if you could switch it to using four-spaces instead of a tab, my eyeballs would appreciate it :grin: